### ISSUE TITLE
[Feature request] ffsend default to upload

### ISSUE THREAD

**Issue by jdorel** [timvisee/ffsend#58]
My usage of `ffsend` is to quickly send archives from the CLI. To that end would it be possible to make `ffsend upload` the default action, so that we can just write `ffsend <file>` to upload the file ?

**Comment by timvisee** [comment]
Cool! `ffsend <file>` could become ambiguous in some cases, that's why I won't implement that directly.

You can however use the `ffput` binary name to achieve this, please refer to this:
https://github.com/timvisee/ffsend/blob/master/README.md#binary-for-each-subcommand-ffput-ffget

If that doesn't work in your environment, you can use the short subcommand `ffsend u <file>` or create an alias.

Does this match the feature you're requesting here?

**Comment by jdorel** [comment]
Yes it does !

Would it be possible to 'install' those binaries during installation (I am using the AUR package) ? I think the best place for those is in /usr/bin, and its best to let the package manager handle this location. 

Also, thank you very much for this tool !

**Comment by timvisee** [comment]
I wonder whether that is a good idea, as some might consider that as cluttering the system. I don't believe there's feature flag support in AUR that would allow me to move such installation options behind a toggle.

Does anybody else have any thoughts on this?

**Comment by iomintz** [comment]
I don't think it should create those by default. Although there is some precedent, as `sl` creates an alias under `LS`. If the user chooses to make these binaries by themselves, they won't be overridden by pacman anyway. Either don't do this at all, or make it a separate package like `ffsend-aliases`. I'm also not sure how helpful installing `ffdel` by default would be anyway.
